examples/jenkins/jenkins-ephemeral-template.json

```diff
@@ -99,6 +101,14 @@
                   {
                     "name": "JENKINS_PASSWORD",
                     "value": "${JENKINS_PASSWORD}"
+                  },
+                  {
+                    "name": "KUBERNETES_MASTER",
```

Comment:
true, but only within the image right? (I can't define env vars in terms of a combination of other env vars in the pod template as far as i know).

so the run script would have to be updated to build up the value.